src/NHibernate/Dialect/Ingres9Dialect.cs

```diff
@@ -65,10 +70,30 @@ public override SqlString GetLimitString(SqlString queryString, SqlString offset
 			return pagingBuilder.ToSqlString();
 		}
 
+		public override string GetSequenceNextValString(string sequenceName)
+		{
+			return "select " + GetSelectSequenceNextValString(sequenceName) + " as seq";
+		}
+
+		public override string GetSelectSequenceNextValString(string sequenceName)
+		{
+			return "next value for " + sequenceName;
+		}
```

Comment:
Hibernate use the alternate syntax: `sequenceName + ".nextval"`. No indication in [Ingres 11 documentation](http://docs.actian.com/ingres/11.0/index.html#page/SQLRef/04_ElementsSQLStatements.04.085.htm) (nor in [10](http://docs.actian.com/ingres/10S/index.html#page/SQLRef/SQLRef_Body.1.0155.htm)) if one of these syntaxes is supported since longer.

I am not sure Hibernate is good source for this case though, since their `GetSequenceNextValString` syntax is not even documented in previous link (they do not call `GetSelectSequenceNextValString`, they return `"select nextval for " + sequenceName`

The way Ingres documentation is written let me think the "main" syntax is the one currently in this PR.